package-releases/base/kustomization.yaml

```diff
@@ -3,8 +3,6 @@ kind: Kustomization
 resources:
   - cronjob.yaml
   - imagestream.yaml
-generatorOptions:
```

Comment:
@pacospace https://github.com/thoth-station/thoth-application/blob/master/package-update/base/kustomization.yaml doesn't contain it, so deleted here as well :smile:

Reply:
Sounds good :D

Reply:
Actually, let's ask @harshad16, not convinced here :)

Reply:
yes, this is not needed, 
it needed when we create configmap or secret with the kustomize file, as we don't in this case, it can be removed.